docs/deprecated_wiki/Configuration-module-14.10.md

```diff
@@ -1,86 +0,0 @@
-# Configuration Module (14.10)
```

Comment:
Based on the version info alone this document is very out of date. It seems to be a manual recreation of some of the settings from the config table. This info can be viewed directly in the SQL by developers. I'm not sure whether we're planning to maintain an ongoing description of all config settings - @ridz1208 ?